geode-core/src/distributedTest/java/org/apache/geode/alerting/AlertingServiceWithoutListenerDistributedTest.java

```diff
@@ -64,8 +65,8 @@
 import org.apache.geode.test.junit.rules.serializable.SerializableTestName;
 
 /**
- * Distributed tests for {@link AlertingService} with {@link DistributedSystemMXBean} in the
- * JMX Manager without any {@code NotificationListener}s.
+ * Distributed tests for {@link org.apache.geode.alerting.AlertingService} with {@link
```

Comment:
Again, does this need to be fully qualified?

Reply:
Same

Reply:
Removed FQN